SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -119,6 +119,8 @@ class SPATIALGDK_API USpatialClassInfoManager : public UObject
 	TArray<float> GetNetCullDistances() const;
 	uint32 GetComponentIdForNetCullDistance(float NetCullDistance) const;
 
+	uint32_t ComputeActorInterestComponentId(AActor* Actor);
```

Comment:
The name ComputeActorInterestComponentId suggests it could either be static or const function.